README.md

```diff
@@ -24,89 +40,67 @@ Or if you want to use the latest Feathr code from GitHub:
 pip install git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project
 ```
 
-## Quick Start
+## Running Feathr Examples
 
-- Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb) to try it out. There is also a companion [quick start guide](./docs/quickstart.md) containing a bit more explanation on the notebook.
-- For more details, read our [documentation](https://linkedin.github.io/feathr/).
+Follow the [quick start Jupyter Notebook](./feathr_project/feathrcli/data/feathr_user_workspace/nyc_driver_demo.ipynb) to try it out. There is also a companion [quick start guide](./docs/quickstart.md) containing a bit more explanation on the notebook.
 
-## Defining Features with Transformation
+## Documentation
 
-```python
-features = [
-    Feature(name="f_trip_distance",                         # Ingest feature data as-is
-            feature_type=FLOAT),
-    Feature(name="f_is_long_trip_distance",
-            feature_type=BOOLEAN,
-            transform="cast_float(trip_distance)>30"),      # SQL-like syntax to transform raw data into feature
-    Feature(name="f_day_of_week",
-            feature_type=INT32,
-            transform="dayofweek(lpep_dropoff_datetime)")   # Provides built-in transformation
-]
-
-anchor = FeatureAnchor(name="request_features",             # Features anchored on same source
-                       source=batch_source,
-                       features=features)
-```
+For more details, read our [documentation](https://linkedin.github.io/feathr/).
```

Comment:
put this just one line under the first paragraph without headers?

Reply:
This is to highlight the documentation (otherwise folks don't notice)

Reply:
![image](https://user-images.githubusercontent.com/1185090/162807861-f0a1801b-68f7-48df-815b-632d918b252a.png)